microGallery_Android/core-ui/src/main/kotlin/studio/lunabee/amicrogallery/android/core/ui/theme/CoreSpacing.kt

```diff
@@ -8,8 +8,7 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.Dp
 
 object CoreSpacing {
-    // Here goes your spacing
-    // val SpacingXs: Dp = 4.dp
+    const val Bottom_row_spacing = 16
```

Comment:
We rarely use snake_case, camelCase is recommended + use `dp` directly as it should only be use that way. 
```suggestion
    val BottomRowSpacing: Dp = 16.dp
```